github/graphql/pkey/local/local.go

```diff
@@ -0,0 +1,30 @@
+package local
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/athenianco/metadata/github/types"
+)
+
+type Local struct {
+	workDir string
+}
+
+func New(workDir string) *Local {
+	return &Local{
+		workDir: workDir,
+	}
+}
+
+func (l *Local) GetPrivateKeyData(ctx context.Context, appId int64) ([]byte, error) {
+	data, err := ioutil.ReadFile(filepath.Join(l.workDir, string(appId)))
+	if err == os.ErrNotExist {
```

Comment:
```suggestion
	if os.IsNotExists(err) {
```
Your code won't work because that error is always wrapped into `os.PathError`.

Reply:
Done.